boards/arm/b_u585i_iot02a/b_u585i_iot02a.dts

```diff
@@ -24,3 +24,34 @@
 		sw0 = &user_button;
 	};
 };
+
+&flash0 {
+	partitions {
+		compatible = "fixed-partitions";
```

Comment:
Let's add a comment specifying that this partition is dedicated to the use of b_u585i_iot02a target with TZEN=0 (so w/o TFM)

Reply:
ok